picard/album.py

```diff
@@ -696,6 +696,21 @@ def column(self, column):
             return self.metadata['~totalalbumtracks']
         elif column == 'discnumber':
             return self.metadata['totaldiscs']
+        elif column == 'covercount':
+            text = ""
+            if self.metadata.images:
+                metadata = self.metadata
+            else:
+                metadata = self.orig_metadata
+
+            number_of_images = len(metadata.images)
+            if getattr(metadata, 'has_common_images', True):
+                text += ngettext("%i image", "%i images",
```

Comment:
Why do you concatenate to empty `text` here? You may just `return ngettext(...)`